gui/src/gdb/client.rs

```diff
@@ -101,12 +101,22 @@ impl<'a, H: GdbHandler> GdbDispatcher for ClientDispatcher<'a, H> {
         let off = res.len();
 
         match_bytes! { data,
+            // https://sourceware.org/gdb/onlinedocs/gdb/Stop-Reply-Packets.html
+            "?" => state.parse_stop_reason(res),
```

Comment:
Better to put a link to `?` document itself too since it is hard to find (I'm one of people who can't find it).